iseq.c

```diff
@@ -188,7 +188,12 @@ rb_iseq_free(const rb_iseq_t *iseq)
             ruby_xfree((void *)body->mark_bits.list);
         }
 
+        if (body->variable.original_iseq != NULL)
+            ruby_xfree(body->variable.original_iseq);
```

Comment:
`ruby_xfree` can handle NULL values, so we don't need to explicitly check that here.

```suggestion
        ruby_xfree(body->variable.original_iseq);
```